FWCore/ParameterSet/python/Config.py

```diff
@@ -554,6 +558,41 @@ def extend(self,other,items=()):
                 newSeq._place(name,self)
         self.__dict__['_Process__InExtendCall'] = False
 
+    # FIXME define explicitly if the items should be referenced or copied
```

Comment:
Would it be easier to maintain if you wrapped a ```cms.Process``` into a class that makes the ```cms.Process``` appear to be a python module? Then you'd just be able to call ```Process.extend```.